src/Depressurizer/Database.cs

```diff
@@ -441,6 +462,26 @@ public SortedSet<string> GetDevelopers(int appId, int depth)
             return result;
         }
 
+        public string getExistsDatabasePath(string custPath = "")
+        {
+            if (custPath != "" && File.Exists(custPath))
+            {
+                return custPath;
+            }
+
+            if (File.Exists(Locations.File.DatabaseGzip))
```

Comment:
Are we going to prefer Gzip version or should this be a user option as default saving strategy?

Reply:
I haven't thought of any necessity for user options, although I have imagined it to be switchable.